README.rst

```diff
@@ -88,7 +88,7 @@ Here's the most direct way to get your work merged into the project.
 1. Fork the project
 #. Clone down your fork
 #. Implement your feature or bug fix and commit changes
-#. If the change fixes a bug at `Red Hat bugzilla <https://bugzilla.redhat.com/>`_, or if it is important to the end user, add the following block to the commit message::
+#. If the change fixes a bug at `Red Hat bugzilla <https://bugzilla.redhat.com/buglist.cgi?component=dnf>`_, or if it is important to the end user, add the following block to the commit message::
```

Comment:
We are actually interested in all bugs related to the commit, not just dnf bugs. Sometimes there is a bug reported to another component (e.g. libdnf or dnf-plugins-core, but it could be anything), but the fix is at least partially in dnf or requires a change in dnf to allow fix elsewhere. In either case, the bug report is a useful information. For that reason, I wouldn't change the bugzilla link in this case.

Reply:
I don't know who to figure out if a bug is related to `dnf`, `libdnf` or `dnf-plugins-core`. Having a joint GitLab instance with labels probably be better for searching for existing bugs and triaging them, but there is no GitLab. I don't mind if you edit PR though, to add the way you do this.

Reply:
The point is, if you know about a resolved or related bug, reference it in the commit message, and If you don't know about any bug, don't reference any bugs. The author of the PR usually knows what they are changing and whether the change is based on a bug report. I just don't want to limit the link only to dnf bugs, because we will appreciate any relevant bugs referenced in the commit messages.

Reply:
What about users who just want to report bugs? They discover this repository, because Google sends them here. Maybe there a problem with managing incoming issues? Then the more problems they have to face, the less will actually report anything.

Reply:
The section "Contribution" is not for users who just want to report bugs.

Reply:
I would submit PRs, but existing queue is already too long https://github.com/rpm-software-management/dnf/pulls/abitrolly so I am not planning to spend more time on spamming this tracker. Just want a direct link to search for existing bugs.

Reply:
And I am fine with changing the link in "Bug reporting" section. I am just saying this section is unrelated to reporting bugs or searching for existing issues, so I don't want the link to be changed here as well.

Reply:
Understood.

Reply:
So you don't want to change the link in "Bug reporting" section either? I thought it was a good idea. But I can open a PR myself if you want.

Reply:
Need to find a job. No time for distractions.